docs/docs/examples/cookbooks/airtrain.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            return AirtrainDocumentDatasetEvent(metadata=dataset_meta)
</pre><p>you can probably just return <code>StopEvent(result=metadata)</code> here and omit the <code>complete_workflow</code> step.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/16332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>done!</p>